packages/blade/src/tokens/global/sizes.ts

```diff
@@ -0,0 +1,79 @@
+export type Sizes = Readonly<{
+  /** 0: 0px */
+  0: 0;
+  /** 25: 1px */
+  25: 1;
+  /** 50: 2px */
+  50: 2;
+  /** 75: 3px */
+  75: 3;
+  /** 100: 4px */
+  100: 4;
+  /** 125: 5px */
+  125: 5;
+  /** 150: 6px */
+  150: 6;
+  /** 200: 8px */
+  200: 8;
+  /** 250: 10px */
+  250: 10;
+  /** 300: 12px */
+  300: 12;
+  /** 400: 16px */
+  400: 16;
+  /** 450: 18px */
+  450: 18;
+  /** 500: 20px */
+  500: 20;
+  /** 600: 24px */
+  600: 24;
+  /** 700: 28px */
+  700: 28;
+  /** 800: 32px */
+  800: 32;
+  /** 900: 36px */
+  900: 36;
+  /** 1000: 40px */
+  1000: 40;
+  /** 1200: 48px */
+  1200: 48;
+  /** 2000: 100px */
+  2000: 100;
+  /** 2050: 120px */
+  2050: 120;
+  /** 4000: 300px */
+  4000: 300;
+  /** 6100: 584px */
+  6100: 584;
+}>;
+
+/**
+ * Size tokens are currently not exposed for users (therefore not available in theme)
+ */
+const sizes: Sizes = {
+  0: 0,
```

Comment:
okay now that i look at it,
* we don't have a logical scale
* we aren't exposing it to consumers

so what if we just use the same key:value pair for scale and then when we have logical scale we'll modify at that time and then expose to consumers? because without logic the internal DX is also going to get hampered that when to use what scale?

Reply:
Done in https://github.com/razorpay/blade/pull/1019/commits/9153397891c6d98bf242e36fe58a9eb464af3902 and https://github.com/razorpay/blade/pull/1019/commits/0cbfce49ee2bf630ee7251b3a815ce56daae1a03